Sources/IO/Misc/PDBReader/example/index.js

```diff
@@ -26,6 +26,7 @@ const sphereActor = vtkActor.newInstance();
 const stickActor = vtkActor.newInstance();
 
 filter.setInputConnection(reader.getOutputPort());
+filter.setHideElement('H');
```

Comment:
maybe, it should be names `hideElements` and when you should call `setHideElements(['H'])` or `setHideElements('H', 'C', 'O')`.

But you need to use the proper macro for the `HideElements`.